src/main/java/org/opendatakit/suitcase/net/LoginTask.java

```diff
@@ -27,8 +30,12 @@ protected Void doInBackground() throws IOException, JSONException {
 
     syncWrapper.reset();
     syncWrapper.init(cloudEndpointInfo);
-    syncWrapper.setPrivilegesInfo();
-
+    if(syncWrapper.isInitialized()) {
```

Comment:
after the init method is called it will be initialized again https://github.com/Shubhamr837/suitcase/blob/c2461b8b74378d34cb189e5f1208eecf47df9188/src/main/java/org/opendatakit/suitcase/net/SyncWrapper.java#L64